firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -175,6 +175,20 @@ class BaseTestCase: XCTestCase {
         }
     }
 
+    func mozWaitForElementToBeHittable(_ element: XCUIElement, timeout: TimeInterval? = TIMEOUT) {
```

Comment:
I suggest that we should ensure the keys are `hittable` before proceeding. Let me test this change against the full functional test suite.